webpack.config.js

```diff
@@ -52,6 +52,17 @@ module.exports = {
                         loader: 'raw-loader'
                     }
                 ]
+            },
+            {
+                test: /.*\.(gif|jpe?g|png)$/,
```

Comment:
Storybookと挙動に差が出るので合わせた方が良いと思います
```suggestion
                test: /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|cur|ani)(\?.*)?$/,
```